### PR TITLE
Start a custom "technology radar" for Artsy

### PR DESCRIPTION
**Pull request by joeyAghion**
For a while we've talked about making shared technology decisions and strategy more visible across Artsy. This is a quick attempt to build a custom "technology radar" for Artsy using [ThoughtWorks' approach and tool](https://www.thoughtworks.com/radar/faq).

There are lots of gaps and I'd love for others to add the significant technologies they're discouraging, evaluating, and encouraging to this framework.

The links should start working after this is merged 🤞, but in the meantime you can interact with a version built with the same data [here](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fgist.githubusercontent.com%2FjoeyAghion%2Fde544d14e32778facd5e96fd154aaff5%2Fraw%2Fartsy_technology_radar.csv). Screenshots:

Some orgs give formal ownership of this process to a technology review committee and reevaluate status every quarter or month. It's interesting to imagine how we might evolve a process like that.

### CODE REVIEW

File: playbooks/technology_radar/data.csv [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Consul,hold,tools,FALSE,
+Jenkins CI/CD,hold,tools,FALSE,
+MongoDB,hold,tools,FALSE,
+Kafka,hold,tools,FALSE,
+Pingdom,hold,tools,TRUE,
+Semaphore,hold,tools,FALSE,
+Travis CI,hold,tools,TRUE,
+Typescript,adopt,languages & frameworks,FALSE,
+React.js,adopt,languages & frameworks,FALSE,
+React Native,adopt,languages & frameworks,FALSE,
```
Comment (ashfurrow) [inline playbooks/technology_radar/data.csv]:
```suggestion
React Native,adopt,languages & frameworks,FALSE,
styled-components,adopt,languages & frameworks,FALSE,
```

**Comment by sweir27** [comment]
Wow, this is so cool!

How do you imagine people should edit this? (at-will? with some kind of process?)

**Comment by joeyAghion** [comment]
Good question about how this should be edited, especially since of these choices may be _controversial_.

I think we were already documenting some of these preferences and communicating others informally, so this just makes that information a little more accessible. (If it inspires debate, great!)

At the moment there are entire technology/technique categories where I don't even know what placeholders to include, or where, so I'd love for others to freely add those. E.g., Relay, Redux, Fastlane, ...

Once we've addressed the remaining "easy" stuff (about which there's broad agreement), any meaningful changes probably _should_ be taken up only very deliberately. Maybe by a practice meeting or RFC?

**Comment by SamRozen** [comment]
Great initiative! Thanks for starting this. This visualization is very powerful.

File: playbooks/technology_radar/data.csv [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name,ring,quadrant,isNew,description
+Babel,adopt,tools,TRUE,"This is the description. You can use basic html such as the <strong>strong tag to emphasise keywords and phrases</strong> and insert <a href=""https://www.thoughtworks.com"">anchor links to documentation and referance material</a>."
```
Comment (alloy) [inline playbooks/technology_radar/data.csv]: When is something new? Babel, Travis, GraphQL etc have all been adopted long before eg OpsGenie.

File: playbooks/technology_radar/data.csv [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Postgresql,adopt,tools,FALSE,
+Sidekiq,adopt,tools,FALSE,
+RabbitMQ,adopt,tools,FALSE,
+Datadog APM,adopt,tools,FALSE,
+CircleCI,adopt,tools,FALSE,
+Cloudflare,adopt,tools,TRUE,
+VSCode pairing,trial,tools,TRUE,
+OpsGenie,trial,tools,TRUE,
+HashiCorp Vault,assess,tools,TRUE,
+Apollo Platform,assess,tools,TRUE,
```
Comment (alloy) [inline playbooks/technology_radar/data.csv]: What’s Apollo Platform? Their paid product?

**Comment by alloy** [comment]
> At the moment there are entire technology/technique categories where I don't even know what placeholders to include, or where, so I'd love for others to freely add those. E.g., Relay, Redux, Fastlane, 

@joeyAghion https://github.com/artsy/README/pull/202

**Pull request merged**